pkgs/by-name/ma/ma/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  tclPackages,
+  fetchurl,
+}:
+
+stdenv.mkDerivation {
+  pname = "ma";
+  version = "11";
+
+  src = fetchurl {
+    url = "http://call-with-current-continuation.org/ma/ma.tar.gz";
```

Comment:
this is probably not a stable url. The hash will change when it updates, breaking builds/reproducibility